client/packages/inventory/src/Stocktake/DetailView/AppBarButtons.tsx

```diff
@@ -26,11 +27,11 @@ interface AppBarButtonProps {
 export const AppBarButtonsComponent: FC<AppBarButtonProps> = ({
   onAddItem,
 }) => {
-  const isDisabled = useStocktake.utils.isDisabled();
   const { OpenButton } = useDetailPanel();
   const t = useTranslation('common');
-  const { data } = useStocktake.document.get();
   const { print, isPrinting } = useReport.utils.print();
+  const { data } = useStocktake.document.get();
+  const isDisabled = !data || isStocktakeDisabled(data);
```

Comment:
the `utils.isDisabled` hook is requesting the document too, so I've consolidated